examples/mobile/src/lib.rs

```diff
@@ -166,14 +166,18 @@ fn setup_music(asset_server: Res<AssetServer>, mut commands: Commands) {
 // Pause audio when app goes into background and resume when it returns.
 // This is handled by the OS on iOS, but not on Android.
 fn handle_lifetime(
-    mut lifetime_events: EventReader<ApplicationLifetime>,
-    music_controller: Query<&AudioSink>,
+    mut lifetime_events: EventReader<AppLifecycle>,
+    music_controller_q: Query<&AudioSink>,
 ) {
+    let Ok(music_controller) = music_controller_q.get_single() else {
+        return;
+    };
+
     for event in lifetime_events.read() {
         match event {
-            ApplicationLifetime::Suspended => music_controller.single().pause(),
-            ApplicationLifetime::Resumed => music_controller.single().play(),
-            ApplicationLifetime::Started => (),
+            AppLifecycle::Suspended => music_controller.pause(),
+            AppLifecycle::Running => music_controller.play(),
+            _ => (),
```

Comment:
would be better to list explicitly the other variants of the enum, as the example is here to demonstrate their existence